src/config/Config.zig

```diff
@@ -5780,6 +5780,27 @@ pub const BackgroundBlur = union(enum) {
     }
 };
 
+/// See window-decoration
+pub const WindowDecoration = enum {
+    true,
+    server,
+    false,
```

Comment:
Same as `true` above, except call it `none`. This matches the protocol more directly.